test-cases/longevity/longevity_d_repo.yaml

```diff
@@ -0,0 +1,39 @@
+test_duration: 360
+
+prepare_write_cmd:  ["cassandra-stress user profile=/tmp/profile-top.yaml no-warmup 'ops(insert=1)' n=35104 -mode cql3 native -rate threads=96"]
+
+stress_cmd: ["cassandra-stress user profile=/tmp/profile-cache.yaml     'ops(readq=1)'  cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native -rate fixed=615/s threads=32   ",
+             "cassandra-stress user profile=/tmp/profile-cold-read.yaml 'ops(readq=1)'  cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native -rate fixed=7329/s threads=96  ",
+             "cassandra-stress user profile=/tmp/profile-write.yaml     'ops(insert=1)' cl=LOCAL_QUORUM no-warmup duration=4h -mode cql3 native -rate fixed=41880/s threads=32 "]
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+round_robin: true
+
+instance_type_db: 'i3en.6xlarge'
+instance_type_loader: 'c5.12xlarge'
+# instance_type_monitor: 'c5.12xlarge'
+# instance_type_db: 'i3en.xlarge'
+
+nemesis_class_name: 'NoOpMonkey'
```

Comment:
We want to have nemesis here at some point.
It's ok to start without for initial testing to make sure it behaves as we expect, but then (before merge) let's change to Sisyphus and test it.

Reply:
Added